interview-questions.md

```diff
@@ -9,59 +9,60 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: this is a shortcut method of recalling elements inside of an object.
 
-  Researched answer:
+  Researched answer: A property of an execution context (global, function or eval) that, in non–strict mode, is always a reference to an object and in strict mode can be any value.
+  
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: React is a framework for Javascript. You use it to make it easier to recall certain functions of code you wrote using constructors.
 
-  Researched answer:
+  Researched answer: React is a free and open-source front-end JavaScript library for building user interfaces or UI components. React can be used as a base in the development of single-page or mobile applications.
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: constructor and props and super.
 
-  Researched answer:
+  Researched answer: Render
 
 
 
 4. What is JSX? What is one notable difference between HTML and JSX?
 
-  Your answer:
+  Your answer: JSX is a a Javascript language that is also embedded in HTML. HTML and JSX use different syntax for writing code.
 
-  Researched answer:
+  Researched answer:  it is a syntax extension to JavaScript. ... it is recommend using it with React to describe what the UI should look like.
 
 
 
 5. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-  Your answer:
+  Your answer: Yarn is a testing application for your code.  Files aren't modified but they are ran for testing. So the app.js file is ran.
 
-  Researched answer:
+  Researched answer: package managers that help to manage a project's dependencies. node modules, yarn lock, package.json.
```

Comment:
Yea, Yarn does a lot of things. Companies are pretty particular about choosing the package managers they use. Hence why Facebook created Yarn in the first place!